### ISSUE TITLE
2.0 Swift版本,关于模型使用struct

### ISSUE THREAD

**Issue by hootigger** [Tencent/wcdb#921]
### The language of WCDB

> e.g. Objective-C, Swift or Java 

Swift

### The version of WCDB

> e.g. v1.0.5
2.0

### The platform of WCDB

> e.g. iOS, macOS or Android
iOS

### The installation of WCDB

> e.g. Cocoapods, Carthage, Maven, AAR Package or Git clone
Cocoapods

### What's the issue?

在旧版中Table声明如下
```
public final class Table<Root: TableCodable>
```

新版(2.0)
变成了
```
public final class Table<Root: AnyObject>
```

原有项目中有大量模型使用struct实现,在迁移升级的时候,遇到 
```
public struct XXXXModel: TableCodable {}

// 会有如下错误, 原因是上面提到的 实现变成了 AnyObject 限制必须是class了
Instance method 'getTable(named:of:)' requires that 'XXXXModel' be a class type
```

综上所述想提问下,底层改用AnyObject 是出于什么考虑?而且Swift也推荐model(不需要继承/引用特性)使用struct实现.
> Post the outputs or screenshots for errors.
> 
> Explain what you want by example or code **in English**.

**Comment by Qiuwen-chen** [comment]
Got it, I'll take change AnyObject to Any in next version in a week or two. Before that, you can directly modify the source code of WCDB to solve the problem.

**Comment by Qiuwen-chen** [comment]
Fixed in wcdb 2.0.1
